### ISSUE TITLE
Size of the training dataset

### ISSUE THREAD

**Issue by sahilg06** [Rudrabha/Wav2Lip#359]
I am training the model on the **crema-d** dataset (the total length of the dataset is about **4 hrs** whereas that of lrs2 is about **29 hrs**).
I also added the data augmentation. but the results are not so good.  Is the size of the dataset an issue or there can be any other problem?
here is the inference of the bazigar video https://drive.google.com/file/d/16bhFVVC7Ou3KRYFdBsUJAZxO8f8pW6cd/view?usp=sharing

**Comment by malayjoshi13** [comment]
yes Sahil, as it's basically a GAN based model thus the more the data, the better will be the results. I am also currently trying to re-train on my custom dataset. If we could connect then we can discuss how to improvise it. Here is my LinkedIn id https://www.linkedin.com/in/malayjoshi13.

**Comment by sd707589** [comment]
@malayjoshi13 Have you solved the training problem? My result video training on my own datasets has the same problem -- still mouth.

**Comment by malayjoshi13** [comment]
Have you trained model from scratch?

**Comment by sd707589** [comment]
> Have you trained model from scratch?

No. I resumed training on the pretrained model. My own training dataset is about 6 hours long of a same person within various clothes.

**Comment by malayjoshi13** [comment]
Please post 1-2 sample videos of how your dataset looks, so that I can undersrand the issue in a better way.

**Comment by sd707589** [comment]
> Please post 1-2 sample videos of how your dataset looks, so that I can undersrand the issue in a better way.

Thanks. This is a sample of my dataset: https://drive.google.com/file/d/1UhmnoTdPIuut7Uj7t-chh8UOMYi0SSGu/view?usp=sharing.
My dataset was downloaded from the website. I divided it by voice and extracted the human face from it.

**Comment by burning846** [comment]
The reason might be that your video has a narrow range of face angles, while LRW dataset has a large range of face angles and face IDs.

**Comment by mrlihellohorld** [comment]
How long is each video in your data set？
